src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1317,6 +1348,167 @@ public void Dispose()
                     disposable?.Dispose();
                 }
             }
+
+            public override TransactionOperationsMerger.IReplayableCommandDto<TransactionOperationsMerger.MergedTransactionCommand> ToDto(JsonOperationContext context)
+            {
+                var buffer = new byte[_totalSize];
+                Marshal.Copy((IntPtr)_buffer, buffer, 0, _totalSize);
+                var strBuffer = Convert.ToBase64String(buffer);
```

Comment:
This may be big, we need to investigate if we can do this in unmanaged memory more easily. 
`Sparrow.Utils.Base64` give you the option to go from `byte*` of buffer to `byte*` of base 64 without allocating managed memory.

Reply:
Issue created
[RavenDB-11852](http://issues.hibernatingrhinos.com/issue/RavenDB-11852)